src/Catalog.API/Catalog.API.http

```diff
@@ -1,10 +1,10 @@
 ﻿@Catalog.API_HostAddress = http://localhost:5222
 
 
-GET {{Catalog.API_HostAddress}}/api/v1/catalog/items
+GET {{Catalog.API_HostAddress}}/api/catalog/items?api-version=1.0
```

Comment:
Should we use a variable to capture the API version here?

```suggestion
GET {{Catalog.API_HostAddress}}/api/catalog/items{{targetVersion}}
```

Reply:
We _can_. I followed the existing theme here. There wasn't a variable for `v{{1}}`, but there certainly can be. It _might_ convenient in this scenario to run each API. In a scenario the services evolve independently over time, it's more likely API versions will be heterogenous and, therefore, a single API version parameter is less useful.

It's a simple change that I'm happy to make if you'd like to see it added.

Reply:
I've added a parameter as requested